tests/rkt_tests.go

```diff
@@ -171,12 +171,13 @@ func expectTimeoutWithOutput(p *gexpect.ExpectSubprocess, searchString string, t
 	return expectCommon(p, searchString, timeout)
 }
 
-func patchTestACI(newFileName string, args ...string) {
+func patchACI(inputFileName, newFileName string, args ...string) {
```

Comment:
This will conflict with the autotools branch as well. I guess the one which gets merged after will have to handle the conflict.